src/calendar.js

```diff
@@ -214,15 +226,16 @@ angular.module('ui.calendar', [])
   .directive('uiCalendar', ['uiCalendarConfig', function(uiCalendarConfig) {
     return {
       restrict: 'A',
-      scope: {eventSources:'=ngModel',calendarWatchEvent: '&'},
+      scope: {eventSources:'=ngModel',calendarWatchEvent: '&', resources: '='},

```

Comment:
To keep the API consistent, we should also add a `calendarWatchResource` binding for custom finger prints of resources